lib/core/main.js

```diff
@@ -41,10 +42,11 @@ async function search(query, options = {}) {
   const page = options.page ? options.page * 10 : 0;
   const use_mobile_ua = Reflect.has(options, 'use_mobile_ua') ? options.use_mobile_ua : true;
   const parse_ads = options.parse_ads || false;
-  const additional_params = options.additional_params || null;
+  const additional_params = options.additional_params || {};
```

Comment:
refactoring: aligned with [other function](https://github.com/LuanRT/google-this/pull/38/files#diff-c8065054c65fe77edfce585a229adcd8d39fbcd0258aa3f4791b4063aa1fda6eR146)